docs/specs/openapi.json

```diff
@@ -6103,7 +6103,7 @@
         "securitySchemes": {
```

Comment:
@maskarb Will this securitySchemes be updated as the basic auth moves to service account auth?

Reply:
I had not thought about that at all, but it will probably need to, yes.